ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiCommandService.scala

```diff
@@ -78,9 +78,11 @@ private[apiserver] final class ApiCommandService private[services] (
     submissionTracker.close()
   }
 
-  private def submitAndWaitInternal(request: SubmitAndWaitRequest)(implicit
+  private def submitAndWaitInternal[T](request: SubmitAndWaitRequest)(
+      f: (Future[Either[TrackedCompletionFailure, CompletionSuccess]], LoggingContext) => Future[T]
+  )(implicit
```

Comment:
Why do we need this change? Is it because we want to use this logging context in the caller?

Reply:
I think something like bd007919e4d868343d37d106887e4a84d703b369 reads better (took the freedom to commit a proposal rewording of this change) Feel free to ignore if you disagree, but let me know wdyt

Reply:
Did you remove this commit? ^